centreon/packages/ui/src/components/Modal/Modal.stories.tsx

```diff
@@ -13,7 +13,10 @@ const meta: Meta<typeof Modal> = {
       options: ['small', 'medium', 'large']
     }
   },
-  component: Modal
+  component: Modal,
+  parameters: {
+    chromatic: { delay: 300 }
+  }
```

Comment:
```suggestion
  component: Modal
```

i assume the chromatic delay was for the loading state? which has been removed

Reply:
Yeah right